packages/adapter-drizzle/src/index.ts

```diff
@@ -100,7 +99,7 @@ import type { Adapter } from "@auth/core/adapters"
  *   userId: text("userId")
  *     .notNull()
  *     .references(() => users.id, { onDelete: "cascade" }),
- *   type: text("type").notNull(),
+ *   type: text("type").$type<AdapterAccount["type"]>().notNull(),
```

Comment:
can use the new `AdapterAccountType` introduced in https://github.com/nextauthjs/next-auth/pull/10832 to avoid circular refernce issues